codespell_lib/data/dictionary.txt

```diff
@@ -11235,7 +11235,7 @@ exeptions->exceptions
 exeqution->execution
 exerbate->exacerbate
 exerbated->exacerbated
-exerciese->exercises
+exerciese->exercise
```

Comment:
Can we add the plural, -ed and -ing variations of this typo too.

Reply:
done in second commit